README.md

```diff
@@ -183,4 +183,5 @@ The allowed filter list is:
 Syntax for using a filter:
 
     ss.fetch_orders(parameters_dict={'order_status': 'shipped', 'page': '2'})
- 
\ No newline at end of file
+ 
+ss.fetch_orders() now returns its request, meaning you can handle its output.
```

Comment:
rather than stating this outright, I think I'd prefer that the example above have `request = ` prepended to it to demonstrate the return

Reply:
Updated the code examples by assigning the method call to a variable. Also referenced the Requests documentation for more information about other perhaps relevant methods and attributes of the Response object.